### ISSUE TITLE
excludePoles in config

### ISSUE THREAD

**Issue by gunrunner20** [jalibu/MMM-RNV#2]
**Is your feature request related to a problem? Please describe.**
Currently all directions are shown for a particular station. This might not be meaningful e.g. if living near a final destination. It would be better then just to see the city-bound connections.

**Describe the solution you'd like**
Introdcution of a config parameter _includePoles[]_ to just show tracks (which are called "poles" by RNV) that are of interest

**Comment by jalibu** [comment]
@gunrunner20 die API liefert irgendwie nur noch maximal 6 Journeys. Auch dann, wenn ich 50 abfrage. kA warum

**Comment by jupadin** [comment]
Ja, das Problem hatte ich auch schon. Auch wird trotz einer Verspätung die Verbindung nach planmäßiger Abfahrt nicht mehr geliefert...
Eine Idee, ob das an der API liegt oder an "uns" ?

**Comment by jalibu** [comment]
@jupadin  liegt an der api. hab es mit postman nachgetestet

**Comment by jupadin** [comment]
Ah, danke. Hatte gerade Postman gestartet 😄
